wdl2cwl/main.py

```diff
@@ -351,7 +344,7 @@ def get_cwl_requirements(self, obj: WDL.Tree.Task) -> List[cwl.ProcessRequiremen
             memory_requirement = None
         if "disks" in runtime:
             with WDLSourceLine(runtime["memory"], ConversionException):
-                outdir_requirement = get_outdir_requirement(
+                outdir_requirement = self.get_outdir_requirement(
```

Comment:
Since the get_outdir_requirement( ) is now a method and not a function outside the class